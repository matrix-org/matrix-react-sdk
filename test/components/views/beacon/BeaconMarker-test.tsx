
import React from 'react';
import { mount } from 'enzyme';

import BeaconMarker from '../../../../src/components/views/beacon/BeaconMarker';

describe('<BeaconMarker />', () => {
    const defaultProps = {};
    const getComponent = (props = {}) =>
        mount(<BeaconMarker {...defaultProps} {...props} />);

    it('renders', () => {
        const component = getComponent();
        expect(component).toBeTruthy();
    });
});
