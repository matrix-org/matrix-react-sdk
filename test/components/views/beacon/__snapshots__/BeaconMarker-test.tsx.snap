// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BeaconMarker /> renders marker when beacon has location 1`] = `
<DocumentFragment>
  <span>
    <div
      class="mx_Marker mx_Username_color8"
      id="!room:server_@alice:server"
    >
      <div
        class="mx_Marker_border"
      >
        <span
          class="_avatar_1o69u_17 mx_BaseAvatar _avatar-imageless_1o69u_60"
          data-color="8"
          data-testid="avatar-img"
          data-type="round"
          role="presentation"
          style="--cpd-avatar-size: 36px;"
          title="@alice:server"
        >
          a
        </span>
      </div>
    </div>
  </span>
</DocumentFragment>
`;
